modules/mono/glue/GodotSharp/GodotSharp/Core/DelegateUtils.cs

```diff
@@ -44,6 +44,34 @@ internal static int DelegateHash(IntPtr delegateGCHandle)
             }
         }
 
+        [UnmanagedCallersOnly]
+        internal static unsafe int GetArgumentCount(IntPtr delegateGCHandle, godot_bool* outIsValid)
+        {
+            try
+            {
+                var @delegate = (Delegate?)GCHandle.FromIntPtr(delegateGCHandle).Target;
+                if (@delegate is null)
+                {
+                    *outIsValid = godot_bool.False;
+                    return 0;
+                }
+                var @method = @delegate.Method;
+                if (@method is null)
+                {
+                    *outIsValid = godot_bool.False;
+                    return 0;
+                }
+                *outIsValid = godot_bool.True;
+                return @method.GetParameters().Length;
```

Comment:
Just use the null conditional operator and we don't have to repeat the null check return.

```suggestion
                var @delegate = (Delegate?)GCHandle.FromIntPtr(delegateGCHandle).Target;
                int? argCount = @delegate?.Method?.GetParameters().Length;
                if (argCount is null)
                {
                    *outIsValid = godot_bool.False;
                    return 0;
                }
                *outIsValid = godot_bool.True;
                return argCount.Value;
```

Reply:
Thank you! Never used these features (and haven't used C# to any real degree in over 10 years)

Will do